lib/diyepw/diyepw/get_wmo_station_location.py

```diff
@@ -0,0 +1,30 @@
+import os as _os
+import pandas as _pd
+
+_this_dir = _os.path.dirname(_os.path.realpath(__file__))
+
+def get_wmo_station_location(wmo_station_number:int):
+    """
+        Get the state, and county associated with a WMO station
+        :param wmo_station_number: The WMO Station ID of the weather station to get location information for
+        :return: Either a dict with fields "state" and "county", or None if no information is available for
+            the passed WMO Station ID
+    """
+    wmo_station_info_filepath = _os.path.join(_this_dir, '..', '..', '..', 'inputs', 'Weather_Stations_by_County.csv')
```

Comment:
needs to be passed in by the user

Reply:
I think the better solution is for this file to be moved into the data directory. There's no reason for the user to define their own set of which weather stations are in which counties.

In any case, @amandadsmith, I am not so sure about this function being part of this project in any case. We don't use it internally anywhere, and it's not part of the workflow of generating AMY EPWs. Do you feel like it makes sense to keep this function as part of DIYEPW?